tests/test_bikesListViews.py

```diff
@@ -135,5 +131,4 @@ def test_post_bikes_list_bad_request_status(self, factory, admin):
         request.headers = headers
 
         response = bikes_list(request)
-
-        assert response.status_code == 404
+        assert response.status_code == 400
```

Comment:
Please stay consistent with status codes and change this one to enum :)

Reply:
ok